src/lib/command_line_tests/command_line_tests.ml

```diff
@@ -0,0 +1,46 @@
+open Core
+open Async
+
+(**
+ * This module tests the basic funtionalities of coda through the cli
+ *)
+
+let%test_module "Command line tests" =
+  ( module struct
+    let kill p =
+      Process.run_exn () ~prog:"kill" ~args:[Pid.to_string @@ Process.pid p]
+
+    let test_background_daemon () =
+      let open Deferred.Let_syntax in
+      let port = 1337 in
+      let%bind daemon_process =
+        Process.create_exn ~prog:"dune"
+          ~args:
+            [ "exec"
+            ; "coda"
+            ; "daemon"
+            ; "--"
+            ; "-background"
+            ; "-client-port"
+            ; sprintf "%d" port ]
+          ()
+      in
```

Comment:
Put a sleep here to make sure the daemon gets a chance to initialize. 100ms should be plenty.

Reply:
Is 100 milliseconds still enough in the presence of the hash chunk table deserialization, which occurs (only) in the daemon?

That deserialization may take 3 seconds.